model/flow/epoch.go

```diff
@@ -72,6 +73,7 @@ type EpochSetup struct {
 	Participants       IdentityList   // all participants of the epoch
 	Assignments        AssignmentList // cluster assignment for the epoch
 	RandomSource       []byte         // source of randomness for epoch-specific setup tasks
+	TargetEndTime      time.Time      // the desired real-world end time for the epoch
```

Comment:
following my previous comment, would suggest to keep the representation of  Unix time 
```suggestion
	TargetEndUnixTime      uint64      // desired real-world end time for the epoch in unix time [seconds]
```